package.json

```diff
@@ -38,22 +38,22 @@
     "test.watch": "yarn test --watch"
   },
   "dependencies": {
-    "@stoplight/types": "^11.0.0",
-    "jsonc-parser": "~2.1.1",
+    "@stoplight/types": "^11.1.1",
+    "jsonc-parser": "~2.2.0",
     "lodash": "^4.17.15",
     "safe-stable-stringify": "^1.1"
   },
   "devDependencies": {
-    "@stoplight/scripts": "^7.0.3",
-    "@types/jest": "^24.0.18",
-    "@types/lodash": "^4.14.138",
-    "@types/node": "^12.7.2",
-    "benchmark": "^2.1.0",
+    "@stoplight/scripts": "^7.0.4",
```

Comment:
Might be worth to setup dependabot? Repos such as json/yaml/markdown are pretty well tested, so there automatic updates should be rather safe.

Reply:
If folks want - I personally don't like dependabot except for our own @stoplight dependencies. Too many PRs all the time.

Reply:
Depends. It works quite well in case of Spectral. We could instrument dependabot to check dependencies weekly or bi-weekly, or even monthly.